modules/services/cloudwatch/README.md

```diff
@@ -0,0 +1,34 @@
+# AWS CloudWatch for monitoring AWS resources
+
+This configures a few standard monitoring metrics for AWS resources along with two
+SNS topics subscribed to OpenCrafts ops@ and urgent@ mailing lists respectively.
```

Comment:
This is a public repo, so we should keep in consideration to make the code generic so other people can use this as well.

Reply:
The intension here was to have common default email subscriptions for all services within OpenCraft using this module.

However, as you rightly pointed out, this is a public repo. Having this here, would not only be redundant for anyone outside OpenCraft who would be using this repo, it could also be potentially very annoying for us as well, as we would get paged, if they forget to override it.

Therefore, I have chosen to completely omit this. The ARN for the alert topics are now part of the output.  Also sample subscription scripts are provided in the readme doc. Anyone using this module can now create their own subscriptions using the exposed ARN.

Thanks for pointing this out.